deps/build.jl

```diff
@@ -0,0 +1,4 @@
+default_dir = get(ENV, "CONDA_JL_HOME", abspath(joinpath(dirname(@__FILE__), "usr")))
+open("deps.jl", "w") do f
+    println(f, "default_dir=\"$(escape_string(default_dir))\"")
+end
```

Comment:
You should only write this file if has changed, in order to prevent Conda from being spuriously recompiled:
```jl
deps = "default_dir=\"$(escape_string(default_dir))\"\n"
if !isfile("deps.jl") || readstring("deps.jl") != deps
    write("deps.jl", deps)
end
```